fastlane/spec/tools_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe Fastlane do
+  describe "Fastlane::TOOLS" do
+    it "lists all the fastlane tools" do
+      expect(Fastlane::TOOLS.count).to be >= 15
```

Comment:
Cool, my main worry is the list gets corrupted but could technically be the right size but I guess thats what PRs are for :)